configure.ac

```diff
@@ -415,60 +468,9 @@ if test "x$matlab_threads" = "xyes" -a "x$enable_threads" != "xyes"; then
   AC_MSG_ERROR([The NFFT Matlab interface with thread support requires the threaded NFFT to be built. Please re-run configure with --enable-openmp.])
 fi
 
-
-################################################################################
-# compiler options
-################################################################################
-
-# Try to choose good compiler options.
-if test "x$ac_test_CFLAGS" != "xset"; then
-  saved_CPPFLAGS="$CPPFLAGS"
-  saved_LDFLAGS="$LDFLAGS"
-  saved_LIBS="$LIBS"
-  CPPFLAGS="$CPPFLAGS $nfft_fftw3_CPPFLAGS"
-  LIBS="$LIBS $fftw3_LIBS"
-  LDFLAGS="$LDFLAGS $nfft_fftw3_LDFLAGS"
-  AX_CC_MAXOPT
-  CPPFLAGS="$saved_CPPFLAGS"
-  LDFLAGS="$saved_LDFLAGS"
-  LIBS="$saved_LIBS"
-fi
-
-# override CFLAGS selection when debugging
-if test "x${enable_debug}" = "xyes"; then
-  CFLAGS="-g"
-#  if test "x$ax_cv_c_compiler_vendor" = "xapple"; then
-#    CFLAGS="$CFLAGS $ax_cv_apple_gcc_archflag"
-#  fi
-fi
-
-# add gcc warnings, in debug/maintainer mode only
-if test "x${enable_debug}" = "xyes" || test "x$USE_MAINTAINER_MODE" = "xyes";
-then
-  if test "x$ac_cv_c_compiler_gnu" = "xyes"; then
-    CFLAGS="$CFLAGS -Wall -W -Wcast-qual -Wpointer-arith -Wcast-align -pedantic"
-    CFLAGS="$CFLAGS -Wno-long-long -Wshadow -Wbad-function-cast -Wwrite-strings"
-    CFLAGS="$CFLAGS -Wstrict-prototypes -Wredundant-decls -Wnested-externs"
-    CFLAGS="$CFLAGS -Wundef -Wconversion -Wmissing-prototypes "
-    CFLAGS="$CFLAGS -Wmissing-declarations"
-  fi
-fi
-
-# option to accept C99
-CFLAGS="$CFLAGS $ac_cv_prog_cc_c99"
-
-# use MinGW implementation of printf
-if test "x${host_os}" = "xmingw32" -o "x${host_os}" = "xmingw64"; then
-  CFLAGS="$CFLAGS -D__USE_MINGW_ANSI_STDIO=1"
-fi  
-
+# Add FFTW preprocessor flags.
 CPPFLAGS="$CPPFLAGS $nfft_fftw3_CPPFLAGS"
 
-# add Matlab CFLAGS
-if test "x$ax_prog_matlab" = "xyes"; then
```

Comment:
I couldn't find any location where we set matlab_CFLAGS, so I have removed this. This is unrelated to the other changes, I just figured it would be confusing to leave in since we only check for Matlab further below. At the same time, I didn't want to move this down there again because the idea is to determine the full CFLAGS here in this section so the check for `libatomic` works ok.

Reply:
This seems to be very old code, it should be ok. I also did not find any location for matlab_CFLAGS.